http/swagger.yml

```diff
@@ -1640,6 +1640,71 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Error"
+  /delete:
+    post:
+      summary: delete time-series data from influxdb
+      requestBody:
+          description: predicate delete request
```

Comment:
General nits regarding the format of summaries and descriptions in the swagger.yml – we now auto-generate API docs in the v2 documentation using this swagger file. Because these docs are now hosted and public facing, all summaries and descriptions should be sentence-cased. Descriptions (except response descriptions) should end with a period. 

Other notes:
- "Time series" should not be hyphenated.
- "influxdb" should be formatted as a proper noun: InfluxDB.